app/views/works/_files_list.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= turbo_frame_tag dom_id(work, 'files_list') do %>
+<div class="files-list mb-5">
+  <table class="table table-sm" id="filesTable">
+    <thead class="files-list-header table-light">
+    <tr>
+      <th scope="col" class="table-title">Files
+        <span class="ms-3">
+          <%= turbo_frame_tag dom_id(work, 'edit_file'), src: edit_button_work_path(work, tag: 'file'), target: '_top' %>
+        </span>
+        <%= link_to work_zip_path(work), class: 'ms-4', data: { turbo: false } do %>
+          <span class="fa-solid fa-download" aria-hidden="true"></span><span class="visually-hidden">Download All Files</span>
+        <% end %>
+      </th>
+      <th scope="col">Description</th>
+      <th scope="col">Hide file</th>
+    </tr>
+    </thead>
+    <tbody>
+      <%= render Works::Show::AttachedFileComponent.with_collection(attached_files.sort_by { |attached_file| attached_file.path.downcase },
```

Comment:
Can these be sorted in the controller?

Reply:
Yes. Moved the sorting to `works_controller.rb`.